packages/runner/src/scanner/page.ts

```diff
@@ -28,12 +28,17 @@ export class Page {
     }
 
     public async goto(url: string): Promise<void> {
-        const gotoUrlPromise = this.puppeteerPage.goto(url, { waitUntil: ['load'] });
+        const gotoUrlPromise = this.puppeteerPage
+            .goto(url, { waitUntil: ['load'] })
+            .then(response => {
+                if (response.headers()['content-type'] !== 'text/html ') {
```

Comment:
>if (response.headers()['content-type'] !== 'text/html ') { [](start = 16, length = 58)

Need to support case when header has move values like Content-Type: text/html; charset=utf-8

Reply:
Seems all header names are lower cased: https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#responseheaders

Reply:
Header name can be like this Content-Type

Reply:
Are we 100% sure that we want to only scan these types of documents?

Reply:
We want to scan only html pages, so text/html is the only content type we want to scan.
Here's a full list of content types:
````
application/EDI-X12   
application/EDIFACT   
application/javascript   
application/octet-stream   
application/ogg   
application/pdf  
application/xhtml+xml   
application/x-shockwave-flash    
application/json  
application/ld+json  
application/xml   
application/zip  
application/x-www-form-urlencoded  
audio/mpeg   
audio/x-ms-wma   
audio/vnd.rn-realaudio   
audio/x-wav   
image/gif   
image/jpeg   
image/png   
image/tiff    
image/vnd.microsoft.icon    
image/x-icon   
image/vnd.djvu   
image/svg+xml    
multipart/mixed    
multipart/alternative   
multipart/related (using by MHTML (HTML mail).)  
multipart/form-data  
text/css    
text/csv    
text/html    
text/javascript (obsolete)    
text/plain    
text/xml    
video/mpeg    
video/mp4    
video/quicktime    
video/x-ms-wmv    
video/x-msvideo    
video/x-flv   
video/webm   
application/vnd.oasis.opendocument.text    
application/vnd.oasis.opendocument.spreadsheet  
application/vnd.oasis.opendocument.presentation   
application/vnd.oasis.opendocument.graphics   
application/vnd.ms-excel    
application/vnd.openxmlformats-officedocument.spreadsheetml.sheet   
application/vnd.ms-powerpoint    
application/vnd.openxmlformats-officedocument.presentationml.presentation    
application/msword   
application/vnd.openxmlformats-officedocument.wordprocessingml.document   
application/vnd.mozilla.xul+xml   
````